homu/main.py

```diff
@@ -45,7 +45,8 @@
     'rollup-': 0,
     'rollup': 1,
     'rollup=maybe': 0,
-    'rollup=never': -1,
+    'rollup=never': -2,
+    'rollup=iffy': -1,
```

Comment:
Hmm, I think this change will turn all the `rollup=never` in the database to `rollup=iffy`.

On the other hand I don't see a way to avoid that. Since we're doing the change anyway, could we switch to having like `-100` for rollup=never and `-50` for rollup=iffy so the problem won't happen in the future, if we add more rollup levels?

Reply:
I feel like it's not a big deal, it's easy enough to migrate when this happens